src/dialogs/generic/show-dialog-box.ts

```diff
@@ -76,12 +79,21 @@ export const showAlertDialog = (
 
 export const showConfirmationDialog = (
   element: HTMLElement,
-  dialogParams: ConfirmationDialogParams
+  dialogParams: DialogParams
 ) =>
   showDialogHelper(element, dialogParams, { confirmation: true }) as Promise<
     boolean
   >;
 
+export const showQuestionDialog = (
+  element: HTMLElement,
+  dialogParams: DialogParams
+) =>
+  showDialogHelper(element, dialogParams, {
+    question: true,
```

Comment:
Instead of this extra property, can't we just pass the `confirmText` and `dismissText`?

Reply:
Yes, I think we can. Originally there was additional logic inside based on this property, but that was in the end not needed.

Reply:
Currently we do not have `hass` available here to call `localize()`. Is there an easy way to get access to it here? I think that was the reason why I went with my approach instead.

Reply:
Add `hass` (or just lokalize) as a argument of `showQuestionDialog`?